src/System.Linq/src/System.Linq.csproj

```diff
@@ -24,7 +24,6 @@
     <Compile Include="..\..\Common\src\System\NotImplemented.cs">
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
-    <Compile Include="Properties\AssemblyInfo.cs" />
```

Comment:
@AlekseyTS added this attribute a few days ago (right before System.Linq got pushed to git). It was necessary to fix similar break caused by the transparency sandwich one level of dependencies up. We need to come up with a better plan (either annotate everything or nothing are the two obvious choices...).